commit/merkleroot/observation.go

```diff
@@ -74,24 +81,20 @@ type Observer interface {
 	// ObserveMerkleRoots computes the merkle roots for the given sequence number ranges
 	ObserveMerkleRoots(ctx context.Context, ranges []plugintypes.ChainRange) []cciptypes.MerkleRootChain
 
-	ObserveTokenPrices(ctx context.Context) []cciptypes.TokenPrice
-
-	ObserveGasPrices(ctx context.Context) []cciptypes.GasPriceChain
-
 	ObserveFChain() map[cciptypes.ChainSelector]int
 }
 
-type ObserverImpl struct {
+type MerkleObserver struct {
```

Comment:
Didn't change the way it works in merkleroot in this PR. Did least invasive way. We can later iterate and maybe remove Observer as is completely if it's not needed.